scripts/docker-integration-tests/query_fanout/warning.sh

```diff
@@ -307,3 +306,24 @@ ATTEMPTS=3 TIMEOUT=1 retry_with_backoff render_carbon 9 14 max_fetch_series_limi
 ATTEMPTS=3 TIMEOUT=1 retry_with_backoff find_carbon 16 remote_store_cluster-c_complete_tags_warning
 ATTEMPTS=3 TIMEOUT=1 retry_with_backoff find_carbon 9 max_fetch_series_limit_applied,remote_store_cluster-c_complete_tags_warning
 
+# NB (arnikola): following tests are unrelated to fanout but there's no good
+# testbed story yet; reported query execution errors will be added here and
+# migrated to a proper testbed when it becomes available.
```

Comment:
Hm can we add common methods like `write_metrics` to just `common.sh` and add these to the `prometheus/test.sh` perhaps?  

Seems a better place than the fanout test perhaps?